services/workerplane.go

```diff
@@ -105,7 +104,7 @@ func CalculateMaxUnavailable(maxUnavailableVal string, numHosts int) (int, error
 		// In case there is only one node and rounding down maxUnvailable percentage led to 0
 		maxUnavailable = 1
 	}
-	logrus.Infof("%v worker nodes can be unavailable at a time", maxUnavailable)
+	logrus.Infof("Parsed value of maxUnavailable: %v", maxUnavailable)
```

Comment:
This seems like we switched this in the previous PR? The current active line is more a debug line, while the first one gives the user info on what the value means. `Parsed value of maxUnavailable` doesn't tell the user what it means.

Reply:
I did this because maxUnavailable can get reset based on inactive hosts later on. Instead now I have separate info log lines when I explicitly set and if needed rest maxUnavailable
https://github.com/rancher/rke/pull/1912/files#diff-b8aac5efb9904be5c8ebe84055a5927fR144
https://github.com/rancher/rke/pull/1912/files#diff-ee57d1a642f3b4ee28fb4929afe13026R69